src/store.js

```diff
@@ -301,6 +301,12 @@ function installModule (store, rootState, path, module, hot) {
 
   // register in namespace map
   if (module.namespaced) {
+    const existedModule = store._modulesNamespaceMap[namespace]
+    if (existedModule) {
+      if (process.env.NODE_ENV !== 'production') {
```

Comment:
Can we merge both `if` statement into one so that uglify will remove entire statement on production build?

Reply:
Yes, thanks.
I have changed that.